mobileapp/src/res/routes.ts

```diff
@@ -5,7 +5,7 @@ const routes = {
   home: 'Home',
   logout: 'Logout',
   drawer: 'Drawer',
-  notificationDetails: 'Notification details',
+  notificationDetails: 'NotificationDetails',
```

Comment:
Nie mam teraz jak odpalić, ale czy to przypadkiem nie jest nazwa wyświetlana w headerze? Chyba taka ze spacją jest lepsza.